src/test/java/com/artipie/asto/StorageSaveAndLoadTest.java

```diff
@@ -80,6 +83,39 @@ void shouldSaveFromMultipleBuffers(final Storage storage) throws Exception {
         );
     }
 
+    @SuppressWarnings("PMD.EmptyCatchBlock")
+    @TestTemplate
+    @Timeout(1)
+    void saveIsAtomic(final Storage storage) throws ExecutionException, InterruptedException {
```

Comment:
@Sammers21 this test is very similar to `shouldNotSavePartial` test we already have. The only difference is that we check situation with previously existing value. I'd suggest to rename this method name `shouldNotOverwriteWithPartial` so similar tests will have similar names